app/views/users/_user.html.haml

```diff
@@ -0,0 +1,28 @@
+%th
+  = link_to user.name, user_path(user.id)
+%td= user.email
+%td
+  - if user.confirmed_at?
+    Yes
+  - else
+    No
```

Comment:
too many lines for such simple piece of logic. At least use a ternary

``` haml
%td= user.confirmed_at? ? 'Yes' : 'No'
```

Or you could leverage `I18n` here.

``` yaml
# config/locales/en.yml
en:
  booleans:
    true: Yes
    false: No
```

Then

``` haml
%td= t(user.confirmed_at?, scope: 'booleans')
```